tests/test_MigrateData.py

```diff
@@ -0,0 +1,1072 @@
+                FOREIGN KEY (team_twitch_alert_id) 
+                REFERENCES TeamInTwitchAlert (team_twitch_alert_id)
+                ON DELETE CASCADE 
+            );"""
+    database_manager.db_execute_commit(sql_create_twitch_alerts_table)
+    database_manager.db_execute_commit(sql_create_user_in_twitch_alert_table)
+    database_manager.db_execute_commit(sql_create_team_in_twitch_alert_table)
+    database_manager.db_execute_commit(sql_create_user_in_twitch_team_table)
+
+    verified_table = """
+            CREATE TABLE IF NOT EXISTS verified_emails (
+            u_id integer NOT NULL,
+            email text NOT NULL,
+            PRIMARY KEY (u_id, email)
+            );"""
+
+    non_verified_table = """
+            CREATE TABLE IF NOT EXISTS non_verified_emails (
+            u_id integer NOT NULL,
+            email text NOT NULL,
+            token text NOT NULL,
+            PRIMARY KEY (token)
+            );"""
+
+    role_table = """
+            CREATE TABLE IF NOT EXISTS roles (
+            s_id integer NOT NULL,
+            r_id integer NOT NULL,
+            email_suffix text NOT NULL,
+            PRIMARY KEY (s_id, r_id, email_suffix),
+            FOREIGN KEY (s_id) REFERENCES GuildExtensions (guild_id)
+            );"""
+
+    re_verify_table = """
+            CREATE TABLE IF NOT EXISTS to_re_verify (
+            u_id integer NOT NULL,
+            r_id text NOT NULL,
+            PRIMARY KEY (u_id, r_id)
+            );"""
+    database_manager.db_execute_commit(verified_table)
+    database_manager.db_execute_commit(non_verified_table)
+    database_manager.db_execute_commit(role_table)
+    database_manager.db_execute_commit(re_verify_table)
+
+    vote_table = """
+            CREATE TABLE IF NOT EXISTS Votes (
+            vote_id integer NOT NULL,
+            author_id integer NOT NULL,
+            guild_id integer NOT NULL,
+            title text NOT NULL,
+            chair_id integer,
+            voice_id integer,
+            end_time float
+            );"""
+
+    role_table = """
+            CREATE TABLE IF NOT EXISTS VoteTargetRoles (
+            vote_id integer NOT NULL,
+            role_id integer NOT NULL
+            );"""
+
+    option_table = """
+            CREATE TABLE IF NOT EXISTS VoteOptions (
+            vote_id integer NOT NULL,
+            opt_id integer NOT NULL,
+            option_title text NOT NULL,
+            option_desc text NOT NULL
+            );"""
+
+    delivered_table = """
+            CREATE TABLE IF NOT EXISTS VoteSent (
+            vote_id integer NOT NULL,
+            vote_receiver_id integer NOT NULL,
+            vote_receiver_message integer NOT NULL
+            );"""
+    database_manager.db_execute_commit(vote_table)
+    database_manager.db_execute_commit(role_table)
+    database_manager.db_execute_commit(option_table)
+    database_manager.db_execute_commit(delivered_table)
+
+    pass
+
+
+def populate_tables():
```

Comment:
only the required data should be populated for each individual test rather than all at once